lib/get/get-outdated-destination-content.js

```diff
@@ -2,6 +2,7 @@ import Promise from 'bluebird'
 import log from 'npmlog'
 
 const BATCH_CHAR_LIMIT = 1990
+const BATCH_SIZE_LIMIT = 100
```

Comment:
It rarely possible to hit max response size if you do `1990/62 = 32,xx`62 being a normal id of an asset/entry unless the user has created entries or assets with on character ids which very unlikely.